axlearn/cloud/gcp/scripts/start_tpu.sh

```diff
@@ -12,6 +12,10 @@ echo "=== AXLearn start_tpu.sh ==="
 # Random sleep to prevent all TPU-VMs overwhelming pypi etc for large slices.
 sleep $((1 + $RANDOM % 30))
 
+# Increase file descriptor limits for `root` to avoid "Too many open files" errors.
```

Comment:
nit: the change the comment to an echo statement so someone reading the logs knows what's going on. That would have helped in catching whether the correct startup script was run as well.

Reply:
Good point! Done.